terraform/modules/forest_node/user-data.tpl

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -euxo pipefail
```
for all bash scripts please
`-e` this exits the script on command failure (rather than you having to check the return code)
'-x` traces commands as they are run (rather than you echoing what you're doing)
'-u` exits on unbound variables
-o pipefail` keeps '-e` for pipes